crates/flox-rust-sdk/src/models/environment.rs

```diff
@@ -11,6 +11,10 @@ use super::{floxmeta, project};
 use crate::providers::git::GitProvider;
 
 pub static CATALOG_JSON: &str = "catalog.json";
+// don't forget to update the man page
+pub static DEFAULT_KEEP_GENERATIONS: usize = 10;
+// don't forget to update the man page
+pub static DEFAULT_MAX_AGE_DAYS: u32 = 90;
```

Comment:
Could make these configurable - not sure if we should do that now or later

Reply:
Ok I'll leave as is for now then?

Reply:
const for now, config later